src/pages/signup/Signup.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./signup.css";
+
+import { Navbar } from "../../components/shared/navbar/Navbar";
+import { Signupform } from "../../components/signupform/Signupform";
```

Comment:
you may use common index file to export all your components from parent folder, it will easy to access